SAMtags.tex

```diff
@@ -80,27 +81,30 @@ \section{Standard tags}
   {\tt MC} & Z & CIGAR string for mate/next segment\\
   {\tt MD} & Z & String for mismatching positions \\
   {\tt MF} & ? & Reserved for backwards compatibility reasons \\
+  {\tt MI} & Z & Molecular-barcode (corrected) sequence or identifier. \\
   {\tt MQ} & i & Mapping quality of the mate/next segment \\
   {\tt NH} & i & Number of reported alignments that contains the query in the current record\\
   {\tt NM} & i & Edit distance to the reference \\
   {\tt OC} & Z & Original CIGAR \\
   {\tt OP} & i & Original mapping position \\
-  {\tt OQ} & Z & Original base quality \\
+  {\tt OQ} & Z & Original base quality.\\
```

Comment:
There is consistency on the **master** branch: no periods in this table (except for one typo on the CT line).  So a minor editorial fix for this PR is to not add periods in its changes to this table.